addons/binding/org.openhab.binding.wifiled/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+	<build>
```

Comment:
You're right.  I only added those lines to be able to compile the project in Intellij IDEA.  I forgot to take take them out before committing.  Or would it be better to leave them in for general compatibility with IDEA?